api/v1alpha1/condition_consts.go

```diff
@@ -52,23 +52,13 @@ const (
 
 // Conditions related to the VirtualMachineImages
 const(
-	// VirtualMachineImageOSTypeSupportedCondition denotes that the OS type in the VirtualMachineImage object is
```

Comment:
Add a comment stating deprecated, but don't remove them (all changes are meant to be additive), and these have been here for a while now. 

Same for the other stuff below.